Week10/DogBreedList.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import {
+  View,
+  Text,
+  FlatList,
+  StyleSheet,
+  TouchableOpacity,
+} from "react-native";
+import axios from "axios";
+import { StackParamList } from "./BreedNav";
+import { useNavigation } from "@react-navigation/native";
+import { StackNavigationProp } from "@react-navigation/stack";
+import { FontAwesome } from "@expo/vector-icons";
+import * as Animatable from "react-native-animatable";
```

Comment:
Use a little caution when using new packages. This library is quite old. I'd stick to using Reanimated.